src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -773,6 +773,55 @@ export default class IconPainter {
     canvas.close();
     canvas.fill();
   }
+
+  /**
+   * This icon is used by `business rule task`.
+   */
+  paintBusinessRuleIcon({ setIconOrigin, c, shape, ratioFromParent, icon }: PaintParameter): void {
```

Comment:
Missing the source of the SVG icon in comment (and in the readme if it's not already here), and in bpmn-support.adoc

Reply:
This icon is made by @alachambre and me from scratch

Reply:
https://twitter.com/adrlach if you want something for the credit

Reply:
And here is the svg source, in case you need it in the futur: https://gofile.io/d/pi6wxt
(I can't add svg file in comment apparently...)

Reply:
Perfect 😄 We don't put credit when it's done from strach. 
But you need to update **bpmn-support.adoc** in order to  inform that the render is done 😉

Reply:
So hard to be an artist nowaday 😢